pkg/minikube/reason/k8s.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reason
+
+import "github.com/blang/semver"
+
+type K8sIssue struct {
+	VersionsAffected map[string]struct{}
```

Comment:
how about it just be a flag list ? 

```
VersionsAffected []string
```

Reply:
yeah I was thinking the same thing, I wanted to be slick with the key lookup but looping is just fine and not that worse runtime wise.